diff --git a/caddy/frankenphp/main.go b/caddy/frankenphp/main.go
index 52a2ee8..87c4601 100644
--- a/caddy/frankenphp/main.go
+++ b/caddy/frankenphp/main.go
@@ -1,6 +1,9 @@
 package main
 
 import (
+	"os"
+	"strings"
+
 	"github.com/caddyserver/caddy/v2"
 	caddycmd "github.com/caddyserver/caddy/v2/cmd"
 	"go.uber.org/automaxprocs/maxprocs"
@@ -15,6 +18,16 @@ import (
 )
 
 func main() {
+	defaultCommand := "php-server"
+	if len(os.Args) < 2 {
+		os.Args = append(os.Args, defaultCommand)
+	} else if os.Args[1] != "--help" && strings.HasPrefix(os.Args[1], "-") {
+		var args []string
+		args = append(args, os.Args[:1]...)
+		args = append(args, defaultCommand)
+		os.Args = append(args, os.Args[1:]...)
+	}
+
 	undo, err := maxprocs.Set()
 	defer undo()
 	if err != nil {
